modules/veteran/app/models/veteran/user.rb

```diff
@@ -20,10 +20,10 @@ def json_data
     end
 
     def bgs_service
-      external_key = @user.common_name || @user.email
+      external_key = [@user.first_name, @user.last_name].join(' ')
```

Comment:
Why not use string interpolation here "#{@user.first_name} #{@user.last_name}"?

Reply:
Changed, want to get it out before the next deploy.